tests/TestCase/Model/Behavior/SearchBehaviorTest.php

```diff
@@ -186,6 +187,32 @@ public function testFinder()
     }
 
     /**
+     * Test the custom "emptyValues" configuration
+     *
+     * @return void
+     */
+    public function testEmptyValues()
+    {
+        $queryString = [
+            'foo' => 'a',
+            'search' => 'b',
+            'group' => false
+        ];
+
+        $this->Articles->behaviors()->get('Search')->config([
+            'emptyValues' => ['a']
+        ]);
+        $query = $this->Articles->find('search', ['search' => $queryString]);
+        $this->assertEquals(2, $query->clause('where')->count());
+
+        $this->Articles->behaviors()->get('Search')->config([
+            'emptyValues' => ['a', false]
+        ]);
+        $query = $this->Articles->find('search', ['search' => $queryString]);
+        $this->assertEquals(1, $query->clause('where')->count());
```

Comment:
assertSame() please to avoid type issues.

Reply:
hmm, I am keeping it consistent with other tests. 
they are all using **assertEquals**

Reply:
No excuse to do it right IMO :)